README.md

```diff
@@ -92,3 +92,119 @@ example below (that starts 4 users, with a spawn rate of 2 per second and runs f
     .venv/bin/locust -f kafka_consumers_locustfile.py --no-web -u 4 -r 2 --run-time 20s --stop-timeout 10
 
 Please consult the locust documentation for details: https://docs.locust.io/en/1.4.3/running-locust-without-web-ui.html
+
+
+### Test Configuration
+
+Locust uses a top level file, a python file to configure the tests.
```

Comment:
You're talking about one specific file here, or..? If yes, can we give its name/path here?

Reply:
Locust gets as a parameter a `locustfile` containing `User` classes, which it discovers, instantiates and uses them for the test.
If I want to run the kafka test I would call:

```
locust kafka_consumers_locustfile.py
```

If I want to use the simple locust file to send sessions I would call:

```
locust simple_locustfile.py
```